app/controllers/media_objects_controller.rb

```diff
@@ -95,8 +95,13 @@ def custom_update
     flash[:notice] = @notice
   end
 
+  def index
+    respond_to do |format|
+      format.json   { render json: MediaObject.all.map{|mo| mo.to_json } }
```

Comment:
I think we need to introduce pagination here and maybe back on collection#index as well.